src/main/webapp/directions.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* exported getDirections */
+
+/**
+ * Gets key information from each leg and displays them on the DOM.
+ */
+function getDirections() {
```

Comment:
Consider adding this to the populateData script.

Reply:
This is a great idea, thanks!

Reply:
`RESOLVED`